opencti-platform/opencti-graphql/src/migrations/1701354091174-files-registration.js

```diff
@@ -0,0 +1,23 @@
+import { executionContext, SYSTEM_USER } from '../utils/access';
+import { rawFilesListing } from '../database/file-storage';
+import { logApp } from '../config/conf';
+import { buildFileDataForIndexing } from '../modules/internal/document/document-domain';
+import { elIndexElements } from '../database/engine';
+import { INDEX_INTERNAL_OBJECTS } from '../database/utils';
+
+export const up = async (next) => {
+  const context = executionContext('migration');
+  logApp.info('[MIGRATION] Starting 1701354091161-files-registration.js');
+  const files = await rawFilesListing(context, SYSTEM_USER, '/', { recursive: true });
+  logApp.info(`[MIGRATION] ${files.length} files to register in index`);
+  const elements = files.map((file) => {
+    return { _index: INDEX_INTERNAL_OBJECTS, ...buildFileDataForIndexing(file) };
+  });
+  await elIndexElements(context, SYSTEM_USER, 'Migration files registration', elements);
```

Comment:
elIndexElements seem to index in bulk, but without any chunk, it looks like it will index everything in one bulk request

Reply:
Yes. One bulk its perfectly fine i think